src/studio/src/designer/backend/Services/Implementation/SourceControlLoggingDecorator.cs

```diff
@@ -422,11 +422,16 @@ private void LogError(Exception ex, string method)
         }
 
         private void LogError(Exception ex, string method, string org, string repository)
+        {
+            LogError(ex, method, org, repository, repository, string.Empty);
+        }
+
+        private void LogError(Exception ex, string method, string org, string repository, string destinationPath, string branch)
         {
             var user = AuthenticationHelper.GetDeveloperUserName(_httpContextAccessor.HttpContext);
             var debugInfo = GetDebugInfo();
 
-            _logger.LogError(ex, "Failed executing method {0} for user {1} in org {2} / repository {3}. Debug info: {4}", method, user, org, repository, debugInfo);
+            _logger.LogError(ex, "Failed executing method {0} for user {1} in org {2} / repository {3}. Destination: {5}. Branch: {5}. Debug info: {6}", method, user, org, repository, destinationPath, branch, debugInfo);
```

Comment:
@tjololo 
Should `Destination` not be arg `4` ?

Reply:
You are correct! Nice catch, I can issue a new PR with a fix

Reply:
In the docs for logging they suggest using the variable name inside of the `{method}` instead of a number `{0}`. 

> Use names for the placeholders, not numbers

https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#log-message-template

Reply:
Even better! I'll make another PR later

Reply:
Fixed in #8303